src/main/java/core/basesyntax/serviceimpl/WriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
+    private static final String REPORT_HEADER = "fruit,quantity";
```

Comment:
Consider moving REPORT_HEADER to a more appropriate place if it's a constant that is relevant to the entire application, such as a Constants class or the Main class, to avoid hardcoding in service implementations.